depthai_sdk/src/depthai_sdk/components/nn_component.py

```diff
@@ -146,6 +146,7 @@ def __init__(self,
         self.image_manip.inputImage.setQueueSize(2)
 
         if isinstance(self._input, CameraComponent):
+            self._change_resize_mode(ResizeMode.LETTERBOX)
```

Comment:
issue here is, that if user sets it later explicitly to eg. crop it will mess everything up (as ImageManip wasn't reset)

Reply:
Removed that